scipy/optimize/_linprog_ip.py

```diff
@@ -1104,8 +1114,8 @@ def _linprog_ip(c, c0, A, b, callback, postsolve_args, maxiter=1000, tol=1e-8,
     valid_permc_spec = ('NATURAL', 'MMD_ATA', 'MMD_AT_PLUS_A', 'COLAMD')
     if permc_spec.upper() not in valid_permc_spec:
         warn("Invalid permc_spec option: '" + str(permc_spec) + "'. "
-             "Acceptable values are 'NATURAL', 'MMD_ATA', 'MMD_AT_PLUS_A', "
-             "and 'COLAMD'. Reverting to default.",
+                                                                "Acceptable values are 'NATURAL', 'MMD_ATA', 'MMD_AT_PLUS_A', "
```

Comment:
Was this intentional?
Let's undo the other whitespace changes, please.

Reply:
Sorry about that. These style changes were unintentional. I will revert these changes